orchestrator/src/main/kotlin/Entrypoint.kt

```diff
@@ -101,6 +102,7 @@ class OrchestratorComponent : EndpointComponent<OrchestratorMessage>(Orchestrato
             is NotifierWorkerResult -> orchestrator.handleNotifierWorkerResult(message.header, payload)
             is NotifierWorkerError -> orchestrator.handleNotifierWorkerError(message.header, payload)
             is WorkerError -> orchestrator.handleWorkerError(message.header, payload)
+            is LostSchedule -> orchestrator.handleLostSchedule(message.header, payload)
```

Comment:
Keep alphabetical order

Reply:
There is no alphabetical order in this construct; the messages are ordered by worker execution.